src/containers/BooksForm.js

```diff
@@ -27,25 +17,68 @@ class BooksForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      id: Math.floor(Math.random() * 1000),
       title: '',
       category: categories[0],
     };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
+    if (e.target.name === 'title') {
+      this.setState({
+        title: e.target.value,
+      });
+    } else if (e.target.name === 'category') {
+      this.setState({
+        category: e.target.value,
+      });
+    }
```

Comment:
You can set this automatically using `name`.
```suggestion
  const name = event.target.name;

  this.setState({
    title: event.target.value
    [name]: event.target.value
  })

```